src/main/java/org/spburegistry/backend/controller/ProjectController.java

```diff
@@ -18,11 +22,15 @@ public class ProjectController {
     @Autowired
     private ProjectService projectService;
 
-    @GetMapping
-    public Iterable<ProjectTO> getAllProjects() {
-        return projectService.findAll();
+    @GetMapping("/projects")
+    public List<ProjectTO> getProjectsBetweenDates(
+    @RequestParam("startDate") @DateTimeFormat(pattern = "yyyy-MM-dd") Date startDate,
```

Comment:
If you delete get request with all project, most likely it should be that your query should output all projects if there are no parameters, but your parameters are required, and nowhere is it checked for their null. To do this, you can use Optional and in service check for null